### PR TITLE
Bump version to v0.28.0 in README and releases.md

### PR DESCRIPTION
**Pull request by vinamra28**
# Changes

Signed-off-by: vinamra28 <jvinamra776@gmail.com>
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```release-note
NONE
```

### CODE REVIEW

File: releases.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.28
+
+- **Latest Release**: [v0.28.0][v0-28-0] (2022-11-25) ([docs][v0-28-0-docs])
+- **Initial Release**: [v0.28.0][v0-28-0] (2022-11-25)
+- **End of Life**: 2023-02-25
```
Comment (vinamra28) [inline releases.md:43]: @tektoncd/cli-maintainers what should be the EOL for this release? Should it be 3 months or more?

**Review by vdemeester — APPROVED** [review]
/lgtm

File: releases.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ Further documentation available:
 
 ## Releases
 
+### v0.28
+
+- **Latest Release**: [v0.28.0][v0-28-0] (2022-11-25) ([docs][v0-28-0-docs])
+- **Initial Release**: [v0.28.0][v0-28-0] (2022-11-25)
+- **End of Life**: 2023-01-24
```
Comment (piyush-garg) [inline releases.md:43]: this should be four months

File: releases.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Tekton CLI produces nightly builds, publicly available on
 
 ### Transition Process
 
-Before release v0.29 Tekton CLI has worked on the basis of an undocumented
+Before release v0.28 Tekton CLI has worked on the basis of an undocumented
```
Comment (piyush-garg) [inline releases.md:23]: this should be fine now

File: releases.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ Tekton CLI produces nightly builds, publicly available on
...
 support period of four months, which will be maintained for the releases between
 v0.26 and v0.27.
```
Comment (piyush-garg) [inline releases.md:25]: make it v0.26 and v0.28

File: releases.md (hunk 2 above)
Comment (vdemeester) [reply releases.md:43]: Do we want to mark 0.28 as an LTS ?

Comment (vinamra28) [reply releases.md:43]: I guess we can start marking LTS releases from January so that they are aligned with Pipelines' LTS releases

**Pull request merged**